src/App.tsx

```diff
@@ -11,14 +11,17 @@ import DashboardT from "./screens/dashboard/DashboardT"
 import Login from './screens/Login'
 import Logout from './screens/Logout'
 import NavBar from "./components/NavBar"
-
+import ChangeTableName from "./screens/dashboard/ChangeTableName";
 function App() {
     return (
         <Router>
             <NavBar/>
 
             <Switch>
                 <Route exact path="/dashboard/tables" component={DashboardT}/>
+                <Route path="/changetablename">
```

Comment:
I would go for something like /table/edit/name if we want to be specific that only name will be changed.